src/main/java/connor/command/AddCommand.java

```diff
@@ -0,0 +1,27 @@
+package connor.command;
+
+import connor.task.TaskType;
+import connor.task.TaskList;
+
+/**
+ * Represents an Add {@code Command}.
+ */
+public class AddCommand extends Command {
+    private TaskType tt;

```

Comment:
Perhaps a bit too general naming for tt?

Reply:
Perhaps, I'll most likely change it in the next push